docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -159,6 +159,9 @@ include::{project-root}/docs/src/test/java/com/google/cloud/spring/autoconfigure
 
 By default, the `SimplePubSubMessageConverter` is used to convert payloads of type `byte[]`, `ByteString`, `ByteBuffer`, and `String` to Pub/Sub messages.
 
+===== Advanced Usage
+You can use message headers to route messages to different Pub/Sub topics. To do so, add the `GcpPubSubHeaders.TOPIC` key and the desired topic name as the value to the message headers. Alternatively, use `PubSubMessageHandler.setTopicExpressionString()` to use a SpEL expression to specify the topic name.
```

Comment:
We try to stick with Spring Team's 1-sentence per line rule. :)

Reply:
Ahh, didn't know that! Duly noted.